modules/charon-core/src/test/java/org/wso2/charon3/core/protocol/endpoints/UserResourceManagerTest.java

```diff
@@ -79,14 +327,17 @@ public void tearDown() {
         String id = user.getId();
 
         return new Object[][]{
-                {id, "userName", null, 200, user}
+                {id, null, null, user},
+                {id, null, "emails", user},
+                {id, "userName,meta", null, user},
+                {id, "userName", "emails", user}
         };
     }
 
     @Test(dataProvider = "dataForGetSuccess")
     public void testGetSuccess(String id, String attributes,
-                               String excludeAttributes, int expectedScimResponseStatus, Object objectUser)
-                               throws CharonException {
+                               String excludeAttributes, Object objectUser)
```

Comment:
can we merge these two lines?

Reply:
Fixed in 6f5ca461fa07ac4b49182320325156a58b80b910